tests/compare-compile-time.fmf

```diff
@@ -0,0 +1,121 @@
+# "big-merge" as the two big competetitors we want to compare in compile
+# time.
+#
+# NOTE: The comparison happens on the same machine for each chroot to reach a
+#       fair comparison at least for each day. Day-to-day comparisons need to
+#       be taken with a pinch of salt!
+#
+# We store all comparison results in the test plans artifact directory (see
+# `TMT_PLAN_DATA` in
+# https://tmt.readthedocs.io/en/stable/overview.html#step-variables).
+# The "results.csv" is the collection of all the information we need when
+# proceeding with drawing diagrams for performance metrics.
+#
+# In order to run this test plan on testing-farm you need to make a request like
+# so:
+#
+#   $ testing-farm request \
+#     --compose Fedora-Rawhide \
+#     --git-url https://github.com/fedora-llvm-team/llvm-snapshots \
+#     --git-ref main \
+#     --arch x86_64 \
+#     --plan /tests/compare-compile-time \
+#     --context distro=fedora-rawhide \
+#     --context arch=x86_64 \
+#     --environment COPR_CHROOT=fedora-rawhide-x86_64 \
+#     --environment YYYYMMDD=$(date +%Y%m%d) \
+#     --environment CONFIG_A=pgo \
+#     --environment CONFIG_B=big-merge \
+#     --environment CONFIG_A_PROJECT_NAME=llvm-snapshots-pgo-$(date +%Y%m%d) \
+#     --environment CONFIG_B_PROJECT_NAME=llvm-snapshots-big-merge-$(date +%Y%m%d).
+#
+# NOTE: This plan was designed to be flexible enough to re-use it for testing
+#       i.e. BOLT performance improvements, once we have a copr repo which
+#       enables it.
+#
+# Docs for this file format: https://tmt.readthedocs.io/en/stable/
+
+summary: LLVM compare compile time
+prepare:
+  # Lower the priority of the testing-farm-tag-repository so that our copr repo is picked up.
+  # See: https://docs.testing-farm.io/Testing%20Farm/0.1/test-environment.html#_tag_repository
+  - name: Set testing-farm-tag-repository priority
+    how: shell
+    script: |
+      if dnf repolist | grep -q testing-farm-tag-repository; then
+        dnf install -y 'dnf5-command(config-manager)' || dnf install -y 'dnf-command(config-manager)'
+        dnf config-manager --save --setopt="testing-farm-tag-repository.priority=999" || \
+          dnf config-manager setopt "testing-farm-tag-repository.priority=999"
+      fi
+
+  - name: Enable llvm-test-suite and compat packages repos from COPR
+    how: shell
+    script:
+      - dnf install -y 'dnf5-command(copr)' || dnf install -y 'dnf-command(copr)'
+      - dnf copr enable -y @fedora-llvm-team/llvm-test-suite $COPR_CHROOT
+      - dnf copr enable -y @fedora-llvm-team/llvm-compat-packages $COPR_CHROOT
+
+  - name: Install dependencies for all OSes
+    how: install
+    package:
+      - cmake
+      - coreutils
+      - git
+      - jq
+      - ninja-build
+      - perf
+      - python3-lit
+      - python3-pip
+      - python3-scipy
+      - python3-setuptools
+      - tcl-devel
+      - tcl
+      - which
+
+      # The following list of packages needed are probably not available on RHEL
```

Comment:
Should this comment start with `TODO:`?

Reply:
Fixed in https://github.com/fedora-llvm-team/llvm-snapshots/pull/1275/commits/b6c63f4876875a43e4f63aced0189b81b54be937